test/test_linting.jl

```diff
@@ -1,15 +1,10 @@
 using PlutoTeachingTools
 using Test
 
-using JuliaFormatter: JuliaFormatter
 using Aqua: Aqua
 using JET: JET
 using ExplicitImports: ExplicitImports
 
-@testset "Code formatting" begin
-    @test JuliaFormatter.format(PlutoTeachingTools; verbose=false, overwrite=false)
-end
-
 @testset "Aqua tests" begin
     Aqua.test_all(PlutoTeachingTools; ambiguities=false)
```

Comment:
```suggestion
    Aqua.test_all(PlutoTeachingTools; ambiguities = false)
```

for testing the format feedback